rbac/api/cross_access/model.py

```diff
@@ -39,6 +39,7 @@ class CrossAccountRequest(models.Model):
     end_date = models.DateTimeField(null=False, blank=False, default=None)
     modified = AutoDateTimeField(default=timezone.now)
     status = models.CharField(max_length=10, default="pending")
+    roles = models.ManyToManyField("management.Role", related_name="cross_requests")
```

Comment:
Just for clarity, since the naming of this model is a bit confusing, maybe we should consider something like `related_name="cross_account_requests"` for the relation?